src/ai/backend/agent/kernel_registry/recovery/recovery.py

```diff
@@ -36,8 +46,8 @@ def __init__(
 
     @classmethod
     def create(cls, args: KernelRegistryRecoveryArgs) -> Self:
-        registry_file_name = f"kernel_registry.{args.agent_id}.dat"
-        fallback_registry_file_name = f"kernel_registry.{args.local_instance_id}.dat"
+        registry_file_name = f"last_registry.{args.agent_id}.dat"
+        fallback_registry_file_name = f"last_registry.{args.local_instance_id}.dat"
```

Comment:
Is there no problem even if the name changes?

Reply:
https://github.com/lablup/backend.ai/blob/b012e5f020401623500ed482e6eb4c1217b436c4/src/ai/backend/agent/agent.py#L3801-L3828
it has never changed and he asked the question because he thought the filename is `kernel_registry.dat`